mungegithub/README.md

```diff
@@ -0,0 +1,109 @@
+
+## Concept
+Mungegitgub uses github label and comment system to communicate with developers and other robots.
+
+### Comment
+Powered by prow plugins and mungegithub, people can add labels to PRs by directly write comment. 
+
+  >  Note: If not necessary, do not add or remove labels from github UI which bypasses Mungegithub access control system.
+
+### Label
+Mungegithub waits for 4 labels:
+* **CLA** "cla-yes" label or "cla-no" label is set automatically. There is a hacker way "cla human-approved" to bypass cla check if it's necessary.
+* **LGTM** "lgtm" is the first level approve, it means "look good to me, but someone else may need to take a look and make final decision". "lgtm" is more like "review: approve" in github-way. Everyone assigned to this pr can say valid "/lgtm", people in github organization can also do it. With prow deployed, people should add "lgtm" label by comment "/lgtm". 
+
+  >  Note: Any code changes after "/lgtm" will automatically remove "lgtm" label
+  
+* **Approve** "approved" is the second level approve, it's more like clicking the "merge" buttom in github-way. So when you are a repo admin or the owner of this part of code, when you actually want to get this pr into master, instead of clicking the merge buttom, simply comment "/approve", Mungegithub will add "approved" label on the pr and when other requirements are satisfied, this pr is going to the submit-queue.
+
+  >  Note: Do not comment "/approve" or add "approved" label unless you are 100% sure you want this change, because after you say that, the pr will be merge any minutes.
+    
+* **Release-note** Release note enforcement is another feather we are seeking from Mungegithub. With template, when prs are create, people should add release note (can be "None") in the pr description. Depended on the release message left, Mungegithub will add "release-note", "release-note-none". If you leave it empty, "release-note-needed" will be added and is going to block merging.
+
+* **do-not-merge** Even with all required parts, you can always have more time by add "do-not-merge" label. 
+
+### OWNERS file
+OWNERS file is the way to organize code owners and write priviliage. There are two parts. Reviews are the people who are suggested to review the pr and approvers are the people who can actually say "/approve" to add the "approve" label. Each path can have its OWNERS file, and this file affects this directory and all subdirectories. More details can be found: [Reviewer and approver](https://github.com/kubernetes/test-infra/tree/master/mungegithub/mungers/approvers)
+
+## Auto-merge a PR with Mungegithub
+
+### Stage One
+
+#### 1. Required CI Required CI status must be green. Due to the lack of github api, Mungegithub gets required CI from configration.
+[labels.png]
+
+#### 2. Four kinds of Labels "cla: yes", "lgtm", "approve", "release-ntoe"/"release-note-none"
+![alt tag](https://github.com/istio/test-infra/blob/mungegithub_doc/mungegithub/images/ci-status.png)
```

Comment:
All these `alt tag` (and those below) have not been rendered properly.

Reply:
The images don't show up correctly since the images themselves are not in istio/test-infra where I pointed. I believe it will fine as we merge this pr.